bin/devtools.dart

```diff
@@ -5,26 +5,36 @@
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:args/args.dart';
 import 'package:http_server/http_server.dart' show VirtualDirectory;
 import 'package:path/path.dart';
-import 'package:args/args.dart';
 
 const argHelp = 'help';
 const argMachine = 'machine';
 const argPort = 'port';
 
 final argParser = new ArgParser()
-  ..addFlag(argHelp, hide: true)
-  ..addFlag(argMachine,
-      negatable: false,
-      abbr: 'm',
-      help: 'Sets output format to JSON for consumption in tools')
-  ..addOption(argPort, abbr: 'p', help: 'Port to serve the web application on');
-
-var machineMode = false;
+  ..addFlag(
+    argHelp,
+    negatable: false,
+    abbr: 'h',
+  )
+  ..addFlag(
+    argMachine,
+    negatable: false,
+    abbr: 'm',
+    help: 'Sets output format to JSON for consumption in tools',
+  )
+  ..addOption(
+    argPort,
+    abbr: 'p',
+    help: 'Port to serve DevTools on',
```

Comment:
Should this say Dart DevTools now?

Reply:
I don't know if we want to call it 'Dart DevTools' everywhere here, or if 'DevTools' is ok for a shorter name. I may keep this PR as is, but doing a pass through and normalizing also works.